_data/releases/0.1/series.yml

```diff
@@ -3,4 +3,5 @@ displayed: false
 hidden: true
 tutorial: false
 connectors:
-  - mysql
+  - id: mysql
+    title: MySQL Connector Plug-in
```

Comment:
Instead of adding this to each series file, can we have one file `_data/connectors.yml`, which contains the id/title pairs of all connectors. Then in the series files, you keep it to the id only, as before. In the template, you resolve the readable name using the id against that connectors file. WDYT?

Reply:
I think its a good approach. Updated accordingly.